Update docs subgraph V2

clemsos opened the pull request:
Really just a change in syntax in the way the graph is called.

How do you think we should mention that we have a v1 / v2 breaking changes? @julien51 @searchableguy @wonderwomancode ?

julien51 commented:
Hum, can you make sure this page https://docs.unlock-protocol.com/tools/subgraph is up to date?

clemsos commented:
> Hum, can you make sure this page https://docs.unlock-protocol.com/tools/subgraph is up to date?

yep, added. Also I added a notice about the V2 on the same page

julien51 commented on docs/tools/subgraph.md:
```diff
@@ -13,22 +13,24 @@ Unlock leverages the [Graph Protocol](https://thegraph.com) to simplify the proc
+[Ethereum Mainnet](https://thegraph.com/explorer/subgraph/unlock-protocol/unlock-v2)
 
-[Rinkeby (Ethereum testnet)](https://thegraph.com/explorer/subgraph/unlock-protocol/unlock-rinkeby)
+[Rinkeby (Ethereum testnet)](https://thegraph.com/explorer/subgraph/unlock-protocol/unlock-rinkeby-v2)
```
```suggestion
```

julien51 commented:
I thought the subgraphs addresses were autogenerated from the network package?

julien51 commented:
I think we are missing Avalanche :/

clemsos commented:
> I think we are missing Avalanche :/

ok added

clemsos commented:
> I thought the subgraphs addresses were autogenerated from the network package?

nope just the networks page so far
